Firestore/core/src/firebase/firestore/util/log_stdio.cc

```diff
@@ -25,8 +25,12 @@ namespace firebase {
 namespace firestore {
 namespace util {
 
+namespace {
+
 LogLevel g_log_level = kLogLevelWarning;
```

Comment:
According to Asan, this was somehow violating ODR, though I'm not sure what symbol it was conflicting with.